docs/adr/004-use-postgresql-for-database.md

```diff
@@ -15,6 +15,15 @@ We will likely need a relational database since the data is mostly structured an
 
 Reference to [different types of databases and their usages](https://www.geeksforgeeks.org/types-of-databases/).
 
+## Options Considered
+
+We consider a few factors when we pick a database:
+- relational
```

Comment:
Nitpick: do we want to expound on a this a bit? Maybe stating that the data is relational, so we should go with a relational database? Just a thought!

Reply:
Added!